### ISSUE TITLE
Self-Supervised Object-in-Gripper Segmentation from Robotic Motions. (arXiv:2002.04487v3 [cs.CV] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#513]
Self-Supervised Object-in-Gripper Segmentation from Robotic Motions. (arXiv:2002.04487v3 [cs.CV] UPDATED)<br>
https://ift.tt/3ldbGOf<br>
<p>Accurate object segmentation is a crucial task in the context of robotic manipulation. However, creating sufficient annotated training data for neural networks is particularly time consuming and often requires manual labeling. To this end, we propose a simple, yet robust solution for learning to segment unknown objects grasped by a robot. Specifically, we exploit motion and temporal cues in RGB video sequences. Using optical flow estimation we first learn to predict segmentation masks of our given manipulator. Then, these annotations are used in combination with motion cues to automatically distinguish between background, manipulator and unknown, grasped object. In contrast to existing systems our approach is fully self-supervised and independent of precise camera calibration, 3D models or potentially imperfect depth data. We perform a thorough comparison with alternative baselines and approaches from literature. The object masks and views are shown to be suitable training data for segmentation networks that generalize to novel environments and also allow for watertight 3D reconstruction.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
